js/skin.js

```diff
@@ -108,7 +108,7 @@ var Skin = React.createClass({
       return;
     }
     if (this.props.controller && this.props.controller.videoVr && this.state.isVrMouseDown) {
-      e.preventDefault();
+      event.preventDefault();
```

Comment:
I think this is a good opportunity to add a unit test for this method. It looks like we don't have good coverage for the VR360 code in skin.js

Reply:
This function is tested already. I have chenged name for param and forgot chenge the param in the place.

Reply:
Just to confirm, is there a unit test for this?

Reply:
@KseniiaSokolovaEpam I'm not seeing unit tests for the `handleVrPlayerMouseDown` function from `skin.js`. We try to always add a unit test on bug fixes since a bug indicates that there's an error condition which could've possibly been avoided with unit tests.

Reply:
@jpfranco ok, thanks for the comment. I added the test for the bug.

Reply:
@KseniiaSokolovaEpam thanks for the update.